### PR TITLE
Hackday sas

### PR DESCRIPTION
**Pull request by SasSwart**

### CODE REVIEW

File: cmd/petstore/petstore.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Using config file "./cmd/petstore/can.yml" as specified.
```
Comment (alexthemayers) [inline cmd/petstore/petstore.go:1]: Sure you want to commit this in?

File: design.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+
```
Comment (alexthemayers) [inline design.md:1]: MIght be prudent to break this out into another directory. Designs should be spread across different files as the collection should grow in the long term

File: openapi/node.go [hunk 3]
```diff
@@ -3,39 +3,73 @@ package openapi
 type Traversable interface {
 	getChildren() map[string]Traversable
 	setChild(i string, t Traversable)
-}
-
-type refContainer interface {
-	Traversable
+	getParent() Traversable
```
Comment (alexthemayers) [inline openapi/node.go:6]: dope

Comment (alexthemayers) [reply openapi/node.go:6]: do you need every node to conform to an interface this size?

**Review by alexthemayers — APPROVED** [review]
Left a few comments within the files, 
Nothing major. Nice changes!

File: cmd/petstore/petstore.go (hunk 1 above)
Comment (SasSwart) [reply cmd/petstore/petstore.go:1]: Removing. Thanks.

File: design.md (hunk 2 above)
Comment (SasSwart) [reply design.md:1]: Good idea. Going to merge for now and lets discuss how to document designs synchronously.

**Pull request merged**